src/ember-app/app/router.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+import config from './config/environment';
+
+const Router = Ember.Router.extend({
+  location: 'hash',
```

Comment:
We use hash routing in this repo since surge.sh (the webserver that hosts single-spa.surge.sh) doesn't support fancy routing.